src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/OrganisationServiceImpl.java

```diff
@@ -151,6 +152,25 @@ public OrganisationsDetailResponse retrieveOrganisations() {
         log.debug("Received new organisation details...");
         return new OrganisationsDetailResponse(organisations);
     }
+
+    @Transactional
```

Comment:
I don't see why you need this here. You are only ever updating a single entity therefore if an exception is thrown before .save or during .save the transactional integrity will always be maintained. Use @Transactional when performing multiple DML statements that all need to succeed or fail together -- not the case here.

Reply:
Removed.